frontends/web/src/components/view/view.module.css

```diff
@@ -248,8 +248,11 @@
     flex-direction: row;
 }
 .dialog .buttons {
-    padding-left: var(--space-default);
-    padding-right: var(--space-default);
+    padding-left: var(--space-half);
+    padding-right: var(--space-half);
+}
+.fit .buttons {
+    margin-top: var(--space-half);
 }
```

Comment:
So far this was only used in Alert component, but with this change we also have dialog + fitcontent and this introduced a small padding but in normal Alert.

Will push a fix.